src/backend/access/external/pxfheaders.c

```diff
@@ -158,6 +166,29 @@ static void add_tuple_desc_httpheader(CHURL_HEADERS headers, Relation rel)
 	pfree(formatter.data);
 }
 
+static void add_projection_desc_httpheader(CHURL_HEADERS headers, ProjectionInfo *projInfo) {
+	int i;
+	char long_number[sizeof(int32) * 8];
+	int *varNumbers = projInfo->pi_varNumbers;
+	StringInfoData formatter;
+	initStringInfo(&formatter);
+
+    /* Convert the number of projection columns to a string */
+    pg_ltoa(list_length(projInfo->pi_targetlist), long_number);
+    churl_headers_append(headers, "X-GP-ATTRS-PROJ", long_number);
+
+	for(i = 0; i < list_length(projInfo->pi_targetlist); i++) {
```

Comment:
@kavinderd What' the code logic of PXF end to parse this header? Does it be included in another pull request?